wikipedia/track.py

```diff
@@ -16,23 +17,33 @@
 QUERY_RULES_ENDPOINT: str = "/_query_rules"
 
 QUERY_CLEAN_REXEXP = regexp = re.compile("[^0-9a-zA-Z]+")
+DEFAULT_SEED = hash(__name__)
```

Comment:
I like the idea of pinning the seed. We're doing something similar in `elastic/logs` through default track parameter value: https://github.com/elastic/rally-tracks/blob/f3e4ad5d1e6a0ece30dd1735c8c77e79ec4b5f1a/elastic/logs/README.md?plain=1#L242 https://github.com/elastic/rally-tracks/blob/f3e4ad5d1e6a0ece30dd1735c8c77e79ec4b5f1a/elastic/logs/track.json#L155

Can we document the default in track's README?